theme/layouts/theme.html.twig

```diff
@@ -47,7 +47,7 @@
         {% endblock %}
     </head>
 
-    {% set defaultFulfillment = (square.store.default_item_fulfillment|split(',')[0])|default('SHIPMENT') %}
+    {% set defaultFulfillment = (store_info.default_item_fulfillment|split(',')[0])|default('SHIPMENT') %}
```

Comment:
Here where do we actually get the store-info schema reference from?

Reply:
Now that i can get it to resolve now let me test it out, I think it works because the "parent" template has access to this

Reply:
yeah, as long as the parent who is extending this file pulls it in, it should be able to access it. i had to update my PR to add `store-info` to all pages since all pages extend this file

Reply:
awesome, thanks!